apps/mobile/src/features/Transactions/useCases/UtxoList/UtxoItem.tsx

```diff
@@ -15,16 +15,16 @@ export const UtxoItem = ({item}: {item: UtxoList[number]['utxos'][number]}) => {
   const {palette: p} = useTheme()
   const {wallet} = useSelectedWallet()
   const {navigateToTxDetails} = useWalletNavigation()
-  const {tokenInfos = new Map<Portfolio.Token.Id, Portfolio.Token.Info>()} =
-    usePortfolioTokenInfos({
-      wallet,
-      tokenIds: Object.keys(item.balance) as Portfolio.Token.Id[],
-      sourceId: 'UtxoList',
-    })
+  const tokenIds = Object.keys(item.balance) as Portfolio.Token.Id[]
```

Comment:
don't see any advantage in this update and it also falls back to casting

Reply:
Object.keys converts them to strings